build.sbt

```diff
@@ -394,7 +388,25 @@ lazy val compilerBridge: Project = (project in internalPath / "compiler-bridge")
     },
     publishLocal := publishLocal.dependsOn(cleanSbtBridge).value,
     altPublishSettings,
-    mimaSettings,
```

Comment:
Please could you leave a comment in code explaining why this isn't MiMa-enforced?